src/news_app/create_feed.js

```diff
@@ -0,0 +1,48 @@
+export const makeFeed = array => {
+    // CREATE NEWS FEED CONTAINER
+    const newsParent = document.querySelector('.news');
+    const newsContainer = document.createElement('div');
+    newsContainer.classList.add('news-list')
+    newsParent.appendChild(newsContainer)
+    const news = document.querySelector('.news-list');
+    // GENERATE NEWS FEED
+    array.forEach(item => {
+        const {title, author, published, description, source, url, urlToImage} = item;
+        // NEW NEWS DIV
+        const newsDiv = document.createElement('div');
+        newsDiv.classList.add('news-item')
+        // ADD IMAGE
+        const newsPic = document.createElement('img');
+        newsPic.setAttribute('width','400px')
+        newsPic.src = urlToImage
+        newsDiv.appendChild(newsPic);
+        // ADD TITLE HEADER
+        const newsHead = document.createElement('h3');
+        newsHead.innerText = title;
+        newsDiv.appendChild(newsHead);
+        // ADD AUTHOR
+        if (author) {
+            const newsAuthor = document.createElement('h4');
+            newsAuthor.innerText = author;
+            newsDiv.appendChild(newsAuthor);
+        }
+        // ADD PUBLISHED DATE
+        const newsPublished = document.createElement('h5');
+        newsPublished.innerText = published;
```

Comment:
I think there is a typo, it is publishedAt not published, then try something with Date, new keyword and some method on this object to get some nice looking date like Tue Nov 16 2021 or something similar